scala/scala.bzl

```diff
@@ -273,6 +368,67 @@ _common_attrs = {
   "jvm_flags": attr.string_list(),
 }
 
+_zinc_compile_attrs = {
+  "_zinc": attr.label(
+      default=Label("@zinc//file"),
+      executable=True,
+      single_file=True,
+      allow_files=True),
+  "_zinc_compiler_jar": attr.label(
+      default=Label("@zinc_0_3_10_SNAPSHOT_jar//jar"),
+      single_file=True,
+      allow_files=True),
+  "_scala_compiler_jar": attr.label(
+      default=Label("@scala_compiler_jar//jar"),
+      single_file=True,
+      allow_files=True),
+  "_incremental_compiler_jar": attr.label(
+      default=Label("@incremental_compiler_0_13_9_jar//jar"),
+      single_file=True,
+      allow_files=True),
+  "_scala_library_jar": attr.label(
+      default=Label("@scala_library_jar//jar"),
+      single_file=True,
+      allow_files=True),
+  "_scala_reflect_jar": attr.label(
+      default=Label("@scala_reflect_jar//jar"),
+      single_file=True,
+      allow_files=True),
+  "_sbt_interface_jar": attr.label(
+      default=Label("@sbt_interface_0_13_9_jar//jar"),
+      single_file=True,
+      allow_files=True),
+  "_compiler_interface_jar": attr.label(
+      default=Label("@compiler_interface_0_13_9_sources_jar//jar"),
+      single_file=True,
+      allow_files=True),
+  "_nailgun_server_jar": attr.label(
+      default=Label("@nailgun_server_0_9_1_jar//jar"),
+      single_file=True,
+      allow_files=True),
+
+}
+
+scala_worker = rule(
+  implementation=_scala_worker_impl,
+  attrs={
+      "main_class": attr.string(),
+      "exports": attr.label_list(allow_files=False),
+      # Worker Args
+      "worker": attr.label(
```

Comment:
is this a private label? Should users ever change this? If private, can we use `_worker`?